### ISSUE TITLE
Is using underlined text where it is NOT a link a failure of WCAG, and if so what SC applies?

### ISSUE THREAD

**Issue by mbgower** [w3c/wcag#4459]
I'm assuming folks would agree that some text in a body of text that appears with an underline underneath it is a problem, primarily because a user may think the underlined content is a text.
I caution about anyone doing this. But my question is: is it actually a WCAG failure?

I'm sharing a screenshot I found in a chat interaction, where some text is underlined to emphasize test phrases that could be used. The text is otherwise the same as other text (no colour difference).

![Image](https://github.com/user-attachments/assets/f79170c1-0249-4e81-a159-51ae5136207b)

Even without added colour, they look like links.

Obviously in a scenario where someone used coloured, underlined text to distinguish links from this black underlined text, that would be a failure of Use of Color. But here, there is no colour.

I'm tempted to try applying it against [Consistent Identification](https://www.w3.org/WAI/WCAG22/Understanding/consistent-identification), but while I _think_ with a bit of creative interpretation I can make the normative language apply, I do not think that was the intended scope of the SC. Thoughts?

**Comment by mraccess77** [comment]
I don’t think it is.  Some style manuals use underlining for titles of works.

Update: to clarify I’m thinking of pre-existing works.

**Comment by mbgower** [comment]
> Some style manuals use underlining for titles of works.

That is a _very_ pre-web convention. On a typewriter, an underline is/was intended to represent italics (in a situation where obviously there was no facility for an italic font face). I have not seen that as a style in an electronic style guide. In the context of web content, many rich text editors don't even offer underline as a separate style anymore.

**Comment by patrickhlauke** [comment]
I would say no, not a failure of 3.2.4. There are no mentions or examples of this (at the time of writing WCAG 2.0 fairly common) scenario in the understanding or techniques. If the intention was to cover this, it was not clear. It seems to focus on text labels and text alternatives, not visual presentation.

As a side note, 3.2.4 has always been a bit...unfocused as an SC...I think in all my years auditing, i've only felt it appropriate to fail something under it a handful of times.

**Comment by mbgower** [comment]
I think we can agree that we don't _want_ people using underline for non-link content?
If so, this something we should be working into some guidance for 3, if we don't think it is covered by 2?
Or is the argument that this is not an a11y issue per se? That if affects all users equally, and is more an example of bad design than accessibility"

**Comment by patrickhlauke** [comment]
This sounds, taking a big step back (away from specific "underline or not" and getting to the core of the issue), more like a WCAG 3 potential issue about visual consistency, visually conveying UICs in a way that they can't be confused with static content, or similar.

**Comment by GreggVan** [comment]
I  am not sure we want underlining to forever forward only be used for links.   Underlining has other semantic meanings and we should not deny them.   Links are usually underlined and color -- but even when not color - underlined text usually looks different than a link - and it is easy to discover it is not a link.   There is no accessibility barrier to underlining things that are not links -- so we have no business making rules about it I don't think.

**Comment by TestPartners** [comment]
You will cause a constitutional crisis in the UK if you prohibit the use of underlines for non-link text. Governments use an underlining convention to convey the degree to which they require their MPs to follow the party line when voting on bills. A "one line whip" is the weakest, while a "three line whip" makes toeing the party line mandatory. They literally underline the text one, two or three times as shown at [https://x.com/alexwickham/status/1127917318586163200](https://x.com/alexwickham/status/1127917318586163200).

As a more general principle, I view the underlining of non-link text as a usability issue that affects everyone. It's essentially the same as when inline links are styled identically to the surrounding non-link text, which we are all agreed is not a WCAG non-conformance.

SC 3.2.4 uses the word "identify" in terms of the label or accessible name, not in terms of what a component looks like. I agree that testers rarely report non-conformances but I think this is because they are just testing what is in front of them rather than thinking about consistency throughout a website. I have seen plenty of websites that use words like Submit, Search, Go and Next randomly for the same function. When user testing a multi-stage hotel booking system, I saw a highly proficient screen reader user completely stumped because each stage contained a Next button, except the final stage, which had a "Book a Room" button. I expect few testers would have reported this.

**Comment by mitchellevan** [comment]
Depending on context, using underlines for both links and non-link text can be a failure of 1.4.1 Use of Color. It's like the problem with bold text shown in #4048.

**Comment by GreggVan** [comment]
Since links are programmatic — it is not just color alone that is used to indicate the link.  so I  do not think it is a failure of 1.4.1  

Gregg

**Comment by patrickhlauke** [comment]
> Since links are programmatic — it is not just color alone that is used to indicate the link.  so I  do not think it is a failure of 1.4.1

"programmatic" or not has no bearing on "visual" success criteria. how is the underlying plumbing relevant to whether or not a user can visually see/distinguish things? please...
